ucsb-cs156-s22/s22-6pm-courses
# s22-6pm-2 Austin Added endpoint to retrieve section info given psid.

# Overview
* Given a personal schedule id, the endpoint returns all the section info for all the classes in the specified personal schedule. 
#10 
* modified tests for the curriculum services, documents, and created tests for the section controller
<img width="1487" alt="Screen Shot 2022-06-01 at 12 14 16 PM" src="https://user-images.githubusercontent.com/47233575/171484060-5e2afcec-9f95-4716-8797-ad11cd22aa26.png">

# Testing
1. Create a personal schedule if you don't already have one.  Take note of the psid and the quarter
2. In another window, get a few valid enrollCd's for that quarter.
3. In swagger, use the `/api/personalcourses/add` endpoint to add two or three courses to that personal schedule
4. Go to the `/api/sections/getsections` endpoint and enter the psId.  You should see JSON that has section information about each of the enrollCds on the personal schedule.
